### PR TITLE
Add checks to avoid PHP Notices in Resources code.

### PR DESCRIPTION
**Pull request by t4k**
Mostly using `isset()`. All original notices are included in the commit messages.

Issue #77

### CODE REVIEW

File: resources/admin/classes/domain/Downtime.php [hunk 1]
```diff
@@ -64,7 +64,7 @@ public function getDowntimeTypesArray() {
 		$result = $this->db->processQuery($query, "assoc");
 		$names = array();
 
-		if (is_array($result[0])) {
+		if (is_array(isset($result[0]))) {
```
[the remark this answers is not in the document]
Comment (t4k) [reply resources/admin/classes/domain/Downtime.php]: Good catch. Thank you. I’ll put in a fix shortly.

**Comment by t4k** [comment]
I’ve changed the `isset()` check to be distinct from the `is_array()` check which should clear things up.

**Comment by jeffnm** [comment]
Everything looks good to me. I think we can merge this.

**Comment by t4k** [comment]
Merging based on the review by @jeffnm.

**Comment by t4k** [comment]
I tried the Squash and Merge option on this PR to try and keep the history cleaner, and it doesn’t create a merge commit, which might be what we want. Maybe we should discuss this in an issue.

**Pull request merged**